src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -31,18 +31,25 @@ public static class ActivatorUtilities
             [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] Type instanceType,
             params object[] parameters)
         {
+            if (provider == null)
```

Comment:
Is there a test that ensures this throws an `ArgumentNullException`?

Reply:
Yes, `TypeActivatorThrowsOnNullProvider`. Moved it over to the right `ActivatorUtilitiesTests` file.